sparkle/src/components/Searchbar.tsx

```diff
@@ -40,9 +51,8 @@ export function Searchbar({
           "s-w-full s-rounded-full s-bg-structure-50 s-font-normal s-placeholder-element-600",
           "s-ring-structure-200 focus:s-ring-action-300",
           "dark:s-ring-structure-300-dark dark:focus:s-ring-action-300-dark",
-          size === "sm" ? "s-text-base" : "s-text-sm",
-          size === "sm" ? "s-py-1.5 s-pl-4 s-pr-8" : "s-py-1 s-pl-4 s-pr-8",
           "s-transition-all s-duration-300 s-ease-out",
+          sizeClasses[size],
```

Comment:
2 things to check if you haven't already
- sm before used to be text-base and now will be text-sm / xs was text-sm and now will be text-xs => check in front that this looks as intended
- uses py instead of pt => also check (but it's more obvious so I guess it's intended)

Reply:
Indeed I checked